tests/init.go

```diff
@@ -72,47 +56,35 @@ var Forks = map[string]*params.ChainConfig{
 	"Istanbul": {
 		ChainID: big.NewInt(1),
 
-		EIP150Block:         big.NewInt(0),
 		EIP155Block:         big.NewInt(0),
-		EIP158Block:         big.NewInt(0),
 		DAOForkBlock:        big.NewInt(0),
 		ByzantiumBlock:      big.NewInt(0),
 		ConstantinopleBlock: big.NewInt(0),
 		PetersburgBlock:     big.NewInt(0),
 		IstanbulBlock:       big.NewInt(0),
 	},
-	"EIP158ToByzantiumAt5": {
-		ChainID:        big.NewInt(1),
-		EIP150Block:    big.NewInt(0),
-		EIP155Block:    big.NewInt(0),
-		EIP158Block:    big.NewInt(0),
-		ByzantiumBlock: big.NewInt(5),
-	},
 	"ByzantiumToConstantinopleAt5": {
 		ChainID: big.NewInt(1),
 
-		EIP150Block:         big.NewInt(0),
-		EIP155Block:         big.NewInt(0),
-		EIP158Block:         big.NewInt(0),
+		EIP155Block: big.NewInt(0),
+
 		ByzantiumBlock:      big.NewInt(0),
 		ConstantinopleBlock: big.NewInt(5),
 	},
 	"ByzantiumToConstantinopleFixAt5": {
 		ChainID: big.NewInt(1),
 
-		EIP150Block:         big.NewInt(0),
-		EIP155Block:         big.NewInt(0),
```

Comment:
Byzantium reference in this commit.

Reply:
I did not remove Byzantium, I removed all references to EIP150